static/src/js/components/KeywordPicker/KeywordPicker.jsx

```diff
@@ -219,14 +220,15 @@ const KeywordPicker = ({
   const handleSubmit = () => {
     const addedKeywords = addKeywords(finalSelectedList.splice(1))
     if (addedKeywords) {
-      value.push(addedKeywords)
+      formData.push(addedKeywords)
```

Comment:
It might be cleaner to start this `handleSubmit` with an early return, rather than having an `else` block with a return. Something like this:

```jsx
const handleSubmit = () => {
    const addedKeywords = addKeywords(finalSelectedList.splice(1))

    if (!addedKeywords) return
    
    // do the rest of the stuff
}
```